src/Locale/en.json

```
@@ -796,6 +796,12 @@
   "map_acronym": "M.A.P.",
   "systolic": "Systolic",
   "diastolic": "Diastolic",
+  "blood_pressure_error": {
+    "missing": "Field is required. Either specify both or clear both.",
+    "exceed": "Value cannot exceed 250 mmHg.",
+    "systolic_less_than_diastolic": "Systolic must be greater than diastolic."
+  },
+  "temperature_error": "Temperature must be between 95°F (35°C) and 106°F (41.1°C).",
```

I'd say, keep this generic as `out_of_range_error` and pass the start and end values as params. That way this error can be reused.

```suggestion
  "out_of_range_error": "Value must be between {{ start }} and {{ end }}.",
```

---

As in the parent component which is `DailyRound.tsx` I am only passing `fah`  value.

---

you can pass like: 

```js
t("out_of_range_error", { start: "95°F (35°C)", end: "106°F (41.1°C)" })
```

---

@rithviknishad  
```js
const start = "95°F (35°C)";
const end = "106°F (41.1°C)";
`Value must be between ${start} and ${end}.`;
```

---

you don't need to do string interpolation. instead refer this and the example I shared above: 

https://www.i18next.com/translation-function/interpolation

---

Done